pkg/transform/reportoutput/html.go

```diff
@@ -99,6 +99,15 @@ func parseTemplates() (*template.Template, error) {
 		"incrementIndex": func(i int) int {
 			return i + 1
 		},
+		"pvcIndex": func() *int {
```

Comment:
PVC index is special because it needs to be mutated. This index doesn't count items in one array, it counts PVCs that come from different arrays.